apps/teacher/src/features/documents/components/DocumentFilesSection.tsx

```
@@ -0,0 +1,181 @@
+      setDownloadingFileId(null);
+    }
+  };
+
+  return (
+    <div className="overflow-hidden rounded-[5px] border border-gray-200 bg-white p-6 md:p-8">
+      <div className="flex flex-col gap-4">
+        {isLoading ? (
+          <div className="space-y-3">
+            {Array.from({ length: 3 }).map((_, index) => (
+              <div
+                key={`admin-document-file-skeleton-${index + 1}`}
```

Biome (2.4.11)

[error] Avoid using the index of an array as key property in an element.

(lint/suspicious/noArrayIndexKey)